### PR TITLE
Update relnotes for 0.11.2003.3107.

### PR DESCRIPTION
**Pull request by cgranade**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/be8840da3a34a98e0647f3d158fac4e203b8043d/articles/relnotes/index.md) | 76 | 42 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a336f67e-b737-4ad8-83cd-d16ac5c685a0_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.11.2003.3107
+
+*Release date: March 31, 2020*
+
+This release contains the following bugfixes for version 0.11.2003.2506.
```
Comment (anpaz-msft) [inline articles/relnotes/index.md:23]: which bugfixes? :)

Comment (efratshabtai) [reply articles/relnotes/index.md:23]: Maybe just say some bug fixes without specifying them?
We don't want to call out the client id specifically.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/efddb939f1a5439e77b197c3967d5617665a2b0e/articles/relnotes/index.md) | 76 | 42 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/30a29db5-6aa6-4d2a-97cd-54c1e9e0dd38_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md (hunk 1 above)
Comment (cgranade) [reply articles/relnotes/index.md]: Updated now.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/c0c1b1edc9a82e5b7fc52eac9ab45efd050597aa/articles/relnotes/index.md) | 76 | 42 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7b24ab33-1c35-46c2-a97a-9fff1134ac7c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**